sdk/servicebus/service-bus/test/README.md

```diff
@@ -62,7 +62,26 @@ Then run `npm run test:node` or `npm run test:browser` from your terminal.
 - Add breakpoints in the test you want to debug. Tip: It is helpful to append the `.only` on the test method if you want only that method to be run.
 - Start Debugging
 
+## Integration tests
```

Comment:
It seems weird that service-bus would be the only one of these to be slightly different and have instructions on how to do AAD authentication. I think there are also a lot more environment variables that it uses in the samples, but maybe I'm misremembering.

Nitpick: it also has the same issue with using `npm install` and not `rush update`.

Reply:
I've tried to generalize them by joining some of what was in this file into the other files that used AAD. Please review this again in context!